content/docs/3.0/tasks/best-practices/pd-scheduling.md

```diff
@@ -0,0 +1,268 @@
+
+PD supports dynamically adding and removing schedulers directly through pd-ctl. For example:
+
+- `scheduler show`: Shows currently running schedulers in the system
+- `scheduler remove balance-leader-scheduler`: Removes (disable) balance-leader-scheduler
+- `scheduler add evict-leader-scheduler-1`: Adds a scheduler to remove all leaders in Store 1
+
+### Add/delete Operators manually
+
+PD also supports adding or removing operators directly through pd-ctl. For example:
+
+- `operator add add-peer 2 5`: Adds peers to Region 2 in Store 5
+- `operator add transfer-leader 2 5`: Migrates the leader of Region 2 to Store 5
+- `operator add split-region 2`: Splits Region 2 into two regions evenly in size
+- `operator remove 2`: Removes currently pending operator in Region 2
+
+### Adjust scheduling parameter
+
+You can check the scheduling configuration using the `config show` command in pd-ctl, and adjust the values using `config set {key} {value}`. Common adjustments include:
+
+- `leader-schedule-limit`: Controls the concurrency of transferring leader scheduling
+- `region-schedule-limit`: Controls the concurrency of adding/deleting peer scheduling
+- `disable-replace-offline-replica`: Determines whether to disable the scheduling to take nodes offline
+- `disable-location-replacement`: Determines whether to disable the scheduling that handles the isolation level of regions
+- `max-snapshot-count`: Controls the maximum concurrency of sending/receiving snapshots for each store
+
+## PD scheduling in common scenarios
+
+This section illustrates the best practices of PD scheduling strategies through several typical scenarios.
+
+### Leaders/regions are not evenly distributed
+
+The rating mechanism of PD determines that leader count and region count of different stores cannot fully reflect the load balancing status. Therefore, it is necessary to confirm whether there is load imbalancing from the actual load of TiKV or storage usage.
+
+Once you have confirmed that leaders/region are not evenly distributed, you need to check the rating of different stores.
+
+If the scores of different stores are close, it means PD mistakenly believes that leaders/regions are evenly distributed. Possible reasons are:
+
+- There are hot regions that cause load imbalancing. In this case, you need to analyze further based on [hot regions scheduling](#hot-regions-are-not-evenly-distributed).
+- There are a large number of empty regions or small regions, which leads to a great difference in the number of leaders in different stores and high pressure on Raftstore. This is the time for a [region merge](#region-merge-is-slow) scheduling.
+- Hardware and software environment varies among stores. You can adjust the values of `leader-weight` and `region-weight` accordingly to control the distribution of leader/region.
+- Other unknown reasons. Still, you can adjust the values of `leader-weight` and `region-weight` to control the distribution of leader/region.
```

Comment:
Perhaps we should note how they adjust these values.

Reply:
You mean `leader-weight` and `region-weight`? If so, they are introduced earlier (### Load balancing). And according to my understanding, they are configured based on the user's specific needs.

Reply:
OK!